app/src/main/java/com/framgia/soundclound/MainActivity.java

```diff
@@ -2,8 +2,15 @@
 
 import android.os.Bundle;
 import android.support.v7.app.AppCompatActivity;
+import android.util.Log;
 
 import com.framgia.soundclound.R;
+import com.framgia.soundclound.data.model.Track;
+import com.framgia.soundclound.data.source.TrackDataSource;
+import com.framgia.soundclound.data.source.TrackRepository;
+import com.framgia.soundclound.data.source.remote.TrackRemoteDataSource;
+
+import java.util.List;
```

Comment:
ERROR: Unused import: java.util.List. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports